public/app/features/dashboard/components/DashboardSettings/DashboardSettings.tsx

```diff
@@ -148,14 +147,7 @@ function getSettingsPages(dashboard: DashboardModel) {
   const permissionsTitle = t('dashboard-settings.permissions.title', 'Permissions');
 
   if (dashboard.id && dashboard.meta.canAdmin) {
-    if (!config.rbacEnabled) {
-      pages.push({
-        title: permissionsTitle,
-        id: 'permissions',
-        icon: 'lock',
-        component: DashboardPermissions,
```

Comment:
I think we can remove `DashboardPermissions` altogether.

Reply:
:+1: will remove it in the next PR

Reply:
tracked in https://github.com/grafana/grafana/issues/76203